libraries/AC_WPNav/AC_Circle.h

```diff
@@ -134,7 +134,8 @@ class AC_Circle
     };
 
     // parameters
-    AP_Float    _radius;        // maximum horizontal speed in cm/s during missions
+    float    _radius;        // radius of circle in cm
```

Comment:
Let's move this definition lower down with the other non-parameters.

nitpick: let's get the alignment of comments fixed too. txs!